test_runner/fixtures/neon_fixtures.py

```diff
@@ -693,7 +694,7 @@ def __init__(self, config: NeonEnvBuilder):
         self.pageservers: List[NeonPageserver] = []
         self.broker = config.broker
         self.pageserver_remote_storage = config.pageserver_remote_storage
-        self.safekeepers_remote_storage = config.sk_remote_storage
+        self.sk_remote_storage = config.sk_remote_storage
```

Comment:
why this naming...?

Reply:
I wanted to make it consistent between `NeonEnv` and `NeonEnvBuilder` (and it was leftovers from another PR); 
I think renaming. `sk_remote_storage` to `safekeepers_remote_storage` everywhere makes sense.